api/src/handlers/server_api.rs

```diff
@@ -83,3 +86,43 @@ impl Handler for StatusHandler {
 		result_to_response(self.get_status())
 	}
 }
+
+/// Convert a SyncStatus to correspond sync_status API string
+fn sync_status_to_api_string(sync_status: SyncStatus) -> String {
```

Comment:
Does it make sense to implement `Display` trait on `SyncStatus` to make it more reusable?